guess-number.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<header>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Guessing Game</title>
+    <img id="office" height="200" width="200"src='https://deltafonts.com/wp-content/uploads/The-Office-tv-logo.png'></img>
+    Guessing Game
+</header>
+
+<body>
+
+    <div id= "instructions">
+
+        <p>What number am I thinking of? You have four opportunities to find a number between 1 and 20. Guess the number correctly and you win!</p>
+    
+    </div>
+
+
+    <label id= "number input"> Guess (1-20) :
+
+        <input id ="guess-input" size= "4" min=1 max="20" > 
+
+        </input>
+    
+    </form>
```

Comment:
This tag doesn't have an opening tag. You didn't need to use a form for this - you'll learn forms further down the line